### ISSUE TITLE
Unable to toggle lights or maked timed lights against monorail stations

### ISSUE THREAD

**Issue by Adreos** [CitiesSkylinesMods/TMPE#1160]
### Describe the problem
Against monorail stations over roads, both the basic one and bus hub one, if an intersection is placed there a timed light cannot be set up, and the mod's use of light toggling cannot be setup. However other aspects of the mod can still be done, such as lane control and intersection properties.

This does not extend to normal monorail track away from the station, all mod properties function there fine.

I tested this on a new map with the only mods enabled being traffic manager and the vanilla mods of unlock all and unlimited money to rule out other mods as the problem.

### Steps to reproduce

1. Build a monorail station
2. Build 2+roads coming off the end to form an intersection.
3. Attempt to build a timed light on that intersection

### Log files

[output_log.txt](https://github.com/CitiesSkylinesMods/TMPE/files/7217742/output_log.txt)

### Savegame?

https://steamcommunity.com/sharedfiles/filedetails/?id=2609333149

### Screenshots?

This busy intersection can only have lanes controlled but not a timed light to properly regulate it
![20210923012050_1](https://user-images.githubusercontent.com/7040403/134511386-190a2ca5-2ee2-49a5-ae33-dffc17d56394.jpg)

### Notes or questions?
Please let me know when the save game is no longer needed so I can remove it from my workshop.

**Comment by krzychu124** [comment]
Thanks for reporting the issue. I also noticed similar behaviour with the metro/train/tram/trolleybus stations. 
The issue is caused by the code for [**Movable Bridge**](https://steamcommunity.com/workshop/filedetails/?id=2496034669) mod compatibility😕 
That mod has own custom logic for controlling traffic on the bridge that is incompatible with TM:PE so timed traffic lights cannot be set on any of those bridges. I need to figure out better solution.

**Comment by krzychu124** [comment]
@Adreos 
I fixed the issue. I'll push an update to [**TM:PE TEST**](https://steamcommunity.com/sharedfiles/filedetails/?id=2489276785) tomorrow. 
If you want to test it locally [here](https://ci.appveyor.com/api/projects/krzychu124/tmpe/artifacts/TMPE.zip?branch=master) you can find the latest version with the fix

**Comment by aubergine10** [comment]
@krzychu124 I can't find the PR for this anywhere...

**Comment by krzychu124** [comment]
> @krzychu124 I can't find the PR for this anywhere...

Yeah, you won't find PR because... I accidentally merged that fix directly to `master` 192258edfb9a23eb4733e56ed6de126cd1d23b6c
